sdv/data_processing/data_processor.py

```diff
@@ -323,138 +323,433 @@ def create_regex_generator(self, column_name, sdtype, column_metadata, is_numeri
 
         return transformer
 
+    def _get_other_transformer(self, kwargs, transformer):
```

Comment:
I think this should be `_get_transformer_instance` mainly because this returns the actual instance of the transformer that will be used, right ?